app/uk/gov/hmrc/trackingconsentfrontend/assets/test/common/getReferrer.spec.ts

```diff
@@ -0,0 +1,21 @@
+import getReferrer from "../../src/common/getReferrer";
+
+describe('getReferrer', () => {
+    let documentReferrer
+    Object.defineProperty(document, 'referrer', { get: () => documentReferrer });
+
+    it('should return just the pathname', () => {
+        documentReferrer = 'https://www.tax.service.gov.uk/another-service'
+
+        const referrer = getReferrer()
+
+        expect(referrer).toEqual('/another-service')
+    })
+
+    it('should return an empty string if no referrer has been set', () => {
+        documentReferrer = ''
+        const referrer = getReferrer()
+
+        expect(referrer).toEqual('')
+    })
```

Comment:
More of a general query about conventions, but why is empty string preferred to null here?
I see there's an `if(referrer)` check elsewhere, so the behavior would be the same... I was just wondering, as I'm not too familiar with conventions that people use for JS/TS.

Reply:
Good question. I guess I was going for consistency with https://developer.mozilla.org/en-US/docs/Web/API/Document/referrer